lambda/recipes-responder/src/main.ts

```diff
@@ -1,19 +1,19 @@
 import {EventType} from "@guardian/content-api-models/crier/event/v1/eventType";
 import {ItemType} from "@guardian/content-api-models/crier/event/v1/itemType";
-//import type {KinesisStreamHandler, KinesisStreamRecord} from "aws-lambda";
 import type {EventBridgeHandler} from "aws-lambda"
 import {registerMetric} from "@recipes-api/cwmetrics";
 import {deserializeEvent} from "@recipes-api/lib/capi";
 import {retrieveIndexData, writeIndexData} from "@recipes-api/lib/recipes-data";
+import {INDEX_JSON, V2_INDEX_JSON} from "./constants"
```

Comment:
These can be imported from the common library (see my earlier point) but must be re-exported from the `index.ts` of the library to do that